jb/itch5/generate_inside.hpp

```diff
@@ -71,12 +70,12 @@ bool record_latency_stats(
  * (before the any output is generated).
  * @returns true if the inside is affected by the change, false otherwise.
  */
-template <typename duration_t>
+template <typename duration_t, typename book_type>
```

Comment:
It is odd. I can see the @tparam book_type in doxygen docs:
https://coryan.github.io/jaybeams/namespacejb_1_1itch5.html#ab989092e95734101159a24c83a54de7c

As well as for bool jb::itch5::record_latency_stats.